pool/app/pool_database/migrations/migration_202408081359.ml

```diff
@@ -17,18 +17,28 @@ let create_changelog_table =
     |sql}
 ;;
 
+(* TODO: Fix manually on staging *)
```

Comment:
Task for issue, not in code. Can be removed.

Reply:
Fixed